index.html

```diff
@@ -42,46 +34,184 @@ <h2 class="title-about">I'm a Software Developer</h2>
+                <ul class="card-lang">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <a href="#" class="btn-works">See Project</a>
+            </div>
+            <div class="card">
+                <h1 class="card-title">Profesional Art Printing Data</h1>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-lang">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>                
+                </ul>
+                <a href="#" class="btn-works">See Project</a>
+            </div>
+            <div class="card">
+                <h1 class="card-title">Profesional Art Printing Data</h1>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-lang">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <a href="#" class="btn-works">See Project</a>
+            </div>
+        </div>
 
     </section>
 
+    
+    <section class="about">
+
+        <section class="introduction">
+            <h1>About me</h1>
         
-   
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+    
+        <a href="#">Get my resume</a>
+        
+        </section>
+
+        
+        
+        <ul class="knowledge">
+            
+            <li class="lang">
+                <h2 class="title-lang"> Language</h2>
+                <ul id="items">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+            </li>
+
+            <li class="frameworks">
+                <h2 class="title"> Framework</h2>
+                <ul id="items">
+                    <li>React.js</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSpec</li>
+                    <li>Capybara</li>
+                </ul>
+            </li>
+
+            <li class="skills">
+                <h2 class="title"> Skills</h2>
+                <ul id="items">
+                    <li>Database Management</li>
+                    <li>Version Control</li>
+                    <li>CLI</li>
+                    <li>API Design</li>
+                    <li>Web Development</li>
+                </ul>
+            </li>
+
+        </ul>
+
+
+
```

Comment:
Great work on the layout! 👏 Just one small improvement is needed: please remove the unnecessary space to ensure a clean and streamlined design. By removing the extra space, you will enhance the overall aesthetic and create a more polished appearance. Keep up the good work! 💪

Reply:
Thanks @shahadat3669, I'll be working on it